docs/api-inference/tasks/image_to_image.md

```diff
@@ -0,0 +1,63 @@
+## Image-to-image
+
+Image-to-image is the task of transforming a source image to match the characteristics of a target image or a target image domain.
+Any image manipulation and enhancement is possible with image to image models.
+
+Use cases heavily depend on the model and the dataset it was trained on, but some common use cases include:
+- Style transfer
+- Image colorization
+- Image super-resolution
+- Image inpainting
+
+<Tip>
+
+For more details about the `image-to-image` task, check out its [dedicated page](https://huggingface.co/tasks/image-to-image)! You will find examples and related materials.
+
+</Tip>
+
+### Recommended models
```

Comment:
As discussed before, we'll need to filter out models that are not warm. The one I would have here is https://huggingface.co/stabilityai/stable-diffusion-xl-refiner-1.0

Reply:
I thought twice about it and I don't know how we can do that. The `inference=warm` status can change over time but since the docs are static once generated, it might not be accurate when a user visits the page.

Reply:
I pushed https://github.com/huggingface/hub-docs/pull/1384/commits/d60825f8fc0fe3fa8ad5877772dac763ce972089 to fetch the inference status of each model. For now I haven't changed the templates for the "Recommended models" part. For `text-to-image` task it would be ok but for `image-to-image` task there are no warm models in the suggested ones.

Reply:
maybe https://huggingface.co/stabilityai/stable-diffusion-xl-refiner-1.0 should be listed on https://huggingface.co/tasks/image-to-image but that seems like a short term solution

Reply:
https://github.com/huggingface/hub-docs/pull/1396/files